analyzers/src/ITs.JsonParser/Json/SarifIssues.cs

```diff
@@ -0,0 +1,54 @@
+﻿namespace ITs.JsonParser.Json;
+
+// Format of JSONs in "output/"
+internal class SarifIssues
+{
+    public Run[] Runs { get; set; }
+
+    public SarifIssue[] AllIssues() => Runs.SelectMany(r => r.Results).ToArray();
+}
+
+internal class Run
+{
+    public SarifIssue[] Results { get; set; }
+}
+
+internal class SarifIssue
+{
+    public string RuleId { get; set; }
+    public string Level { get; set; }
+    public string Message { get; set; }
+    public LocationWrapper[] Locations { get; set; }
+
+    public Location Location => Locations is { Length: > 0 } ? Locations[0].ResultFile : null;
+
+    public object Order() =>
+        Location is not null
```

Comment:
Should use positive condition and swap branches
```suggestion
        Location is null
```